thavalon-webapp/src/components/gameComponents/roleInformation.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useEffect, useState } from "react";
+import { GameSocket, InboundMessage, InboundMessageType, OutboundMessageType } from "../../utils/GameSocket";
+import "../../styles/gameStyles/roleInformation.scss";
+
+interface RoleInfo {
+    abilities: string,
+    assassinatable: boolean,
+    description: string,
+    isAssassin: boolean,
+    otherInfo: string,
+    priorityTarget: string | undefined,
+    role: string,
+    seenPlayers: string[],
+    team: string,
+    teamMembers: string[],
+}
+
+interface Snapshot {
+    roleInfo: RoleInfo,
+    missions: string[],
+    log: string[],
```

Comment:
This’ll be an array of game messages, it should match the ones sent over the websocket